### ISSUE TITLE
Alternative to `len`

### ISSUE THREAD

**Issue by mantielero** [treeform/vmath#70]
Sorry for this probably naive question. I was trying the following:
```nim
import vmath
var a = vec2(10.0,2.0)
echo a.len
```
This works if compiled with `-d:vmathArrayBased`, but how can I get the length of the vector when is not based in arrays?

**Comment by treeform** [comment]
You can use the length() function to get the magnitude:
https://registry.khronos.org/OpenGL-Refpages/gl4/html/length.xhtml

Or do want to get how many dementions the vector has? In this case 2 because its a vec2?

**Comment by mantielero** [comment]
Yes, I meant getting the vector dimension (2 for Vec2). I think it would make sense having an equivalent `len` for the cases where is not compiled with `-d:vmathArrayBased`.

**Comment by treeform** [comment]
Thank you for this issue. I actually have a need for this feature, and so I added it here: https://github.com/treeform/vmath/pull/87
